crates/flightrepl/src/config.rs

```diff
@@ -0,0 +1,76 @@
+pub type GenericError = Box<dyn std::error::Error + Send + Sync>;
+
+#[cfg(target_family = "unix")]
+fn get_os_release() -> Result<String, GenericError> {
```

Comment:
Should this be in `crates/flightrepl`? I don't know if we have anything more suitable.

Reply:
I don't think there's anything else in the OSS repo that needs to supply/generate a user agent.